### DISCUSSION TITLE
Lorem Ipsum extension not working in v2

### DISCUSSION THREAD

**Discussion by pixelsendots** [brackets-cont/brackets#141] [Feature Request]
Seems the Lorem Ipsum extension by Lance Cambell is no longer working properly in v2 of Brackets. Cmd/Ctrl-Shift-L doesn't work anymore. Anyone with the same issue?

Can a Lorem Issue functionality be added to v2?

**Comment by charlypa** [comment]
@pixelsendots 
 Lorem Ipsum extension seems independently maintained.
informed this issue to the repo.
https://github.com/lkcampbell/brackets-lorem-ipsum/issues/17

**Reply by rkii** [reply]
I just noticed it worked fine for me with v2, but after I disabled emmet it stopped working. Re-enabled emmet, and it works fine again.

(I hope this was the right place to comment, if not I apologize, still learning how this works.)
